libs/components/errors/src/lib/modules/error/error.component.html

```diff
@@ -1,34 +1,31 @@
 <div class="sky-error">
-  <div
-    *ngIf="showImage"
-    class="sky-error-image-container sky-margin-stacked-lg"
-  >
-    <div
-      *ngIf="errorType"
-      class="sky-error-image-svg"
-      [ngClass]="['sky-error-' + errorType + '-image']"
-    ></div>
-    <ng-content select="sky-error-image" />
-  </div>
+  @if (showImage) {
+    <div class="sky-error-image-container sky-margin-stacked-lg">
+      @if (errorType) {
+        <div
+          class="sky-error-image-svg"
+          [ngClass]="['sky-error-' + errorType + '-image']"
+        ></div>
+      }
+      <ng-content select="sky-error-image" />
+    </div>
+  }
 
   <div class="sky-error-title sky-font-heading-1 sky-margin-stacked-lg">
-    <ng-container
-      *ngIf="defaultTitle && (errorSvc.replaceDefaultTitle | async) === false"
-      >{{ defaultTitle }}</ng-container
-    >
+    <!-- prettier-ignore -->
```

Comment:
If you wrap the `{{ defaultTitle }}` in a `span` (similar to the block below), do your unit tests pass? If not, maybe you could use the `skyTrim` directive. Either option would be preferrable to overriding the formatter to better communicate the intent of the code: "text should be inline and not include trailing whitespace".